pages/upload.js

```diff
@@ -26,6 +27,22 @@ export default AuthRequired(PERMS, class extends React.Component {
       unfurl: true,
       metadata: null
     };
+
+    try {
+      // Listen for all image paste events on a page
+      PasteImage.on('paste-image', image => {
+          this.handlePaste(image);
+      });
+    } catch (error) {
+        console.log("error", error);
```

Comment:
We should probably let this trickle up, and handle errors more globally than a console.log.